infernity_common_v01.patch

```diff
@@ -19,28 +23,35 @@ index 5ff803df..cd553a42 100644
 +	}
 +}
 +
-+int CalculateTextWidth(const char* s)
-+{
++int CalculateTextWidth(const char* s) {
 +	int l = 0;
 +	while (*s) {
 +		l += fontkern[fontframe[gbFontTransTbl[*s++]]] + 1;
 +	}
 +	return l;
 +}
 +
- #define GG_SWAP(A, B) \
- 	{                 \
- 		(A) ^= (B);   \
+ /**
+  * @brief Draw a line on the back buffer
+  * @param x0 Back buffer coordinate
+@@ -838,6 +855,7 @@ DWORD LoadFileWithMem(const char *pszName, void *p)
+ 
+ /**
+  * @brief Apply the color swaps to a CL2 sprite
++
+  * @param p CL2 buffer
+  * @param ttbl Palette translation table
+  * @param nCel Frame number in CL2 file
 diff --git a/Source/engine.h b/Source/engine.h
-index 3255c778..025f60ee 100644
+index 3255c778..c69ef4fb 100644
 --- a/Source/engine.h
 +++ b/Source/engine.h
-@@ -56,6 +56,8 @@ void CelDrawLightRedSafe(int sx, int sy, BYTE *pCelBuff, int nCel, int nWidth, c
- void CelBlitWidth(BYTE *pBuff, int x, int y, int wdt, BYTE *pCelBuff, int nCel, int nWidth);
+@@ -57,6 +57,8 @@ void CelBlitWidth(BYTE *pBuff, int x, int y, int wdt, BYTE *pCelBuff, int nCel,
  void CelBlitOutline(char col, int sx, int sy, BYTE *pCelBuff, int nCel, int nWidth);
  void ENG_set_pixel(int sx, int sy, BYTE col);
+ void engine_draw_pixel(int sx, int sy);
 +void DrawSolidRectangle(int x0, int dx, int y0, int dy, int color);

```

Comment:
Please change the order of the definitions so that the new ones are before the `engine_draw_pixels` line to minimize the changes to the patch.

Reply:
@Manuel-K you know you can make code suggestion, and even apply them because it's a PR on your branch?

Reply:
What is this evil magic? I still consider multi-part comments a new thing.